shrike/src/main/java/com/ibm/wala/shrike/shrikeCT/ClassReader.java

```diff
@@ -68,7 +68,7 @@ private void parse() throws InvalidClassFileException {
       throw new InvalidClassFileException(offset, "bad magic number: " + magic);
     }
     // Support class files up through JDK 23 (version 67)
```

Comment:
Is this comment still accurate?

Reply:
Good catch!  Fixed in 01e4d98eed53a608b63fba924c0e41a2cb0ce7b5